cmd/ndm_daemonset/filter/osdiskexcludefilter.go

```diff
@@ -103,8 +103,7 @@ func (odf *oSDiskExcludeFilter) Start() {
 		if devPath, err := mountPointUtil.GetDiskPath(); err != nil {
 			klog.Error(err)
 		} else {
-			odf.excludeDevPath = devPath
-			return
+			odf.excludeDevPaths = append(odf.excludeDevPaths, devPath)
 		}
 	}
 	klog.Error("unable to apply os disk filter")
```

Comment:
This message will be printed irrespective of whether the filter configuration succeeded or not. It should be printed only when the filter configuration fails.